quail/processes/wps_climdex_su.py

```diff
@@ -0,0 +1,149 @@
+class ClimdexSU(Process):
+    """
+    Takes a climdexInput object as input and computes the SU
+    (summer days) climdexindex:  that is,  the annual count of days where
+    daily maximum temperature exceeds 25 degreesCelsius
+    """
+
+    def __init__(self):
+        self.status_percentage_steps = dict(
+            common_status_percentages,
+            **{
+                "build_rdata": 90,
+            },
+        )
+        inputs = [
+            ComplexInput(
+                "climdex_input",
+                "climdexInput",
+                abstract="Rdata (.rda) file containing R Object of type climdexInput",
+                supported_formats=[
+                    Format("application/x-gzip", extension=".rda", encoding="base64")
+                ],
+            ),
+            LiteralInput(
+                "ci_name",
+                "climdexInput name",
+                abstract="Name of the climdexInput obejct",
+                min_occurs=1,
+                max_occurs=1,
+                data_type="string",
+            ),
+            LiteralInput(
+                "output_path",
+                "Output file name",
+                abstract="Filename to store the count of days where tmax > 25 degC for each year (extension .rda)",
+                data_type="string",
+            ),
+            log_level,
+        ]
+
+        outputs = [
+            ComplexOutput(
+                "summer_days_file",
+                "Summer days output file",
+                abstract="A vector containing the number of summer days for each year",
+                supported_formats=[
+                    Format("application/x-gzip", extension=".rda", encoding="base64")
+                ],
+            ),
+        ]
+
+        super(ClimdexSU, self).__init__(
+            self._handler,
+            identifier="climdex_su",
+            title="Climdex Summer Days",
+            abstract="Computes the annual count of days where daily maximum temperature exceeds 25 degrees Celsius",
+            metadata=[
+                Metadata("NetCDF processing"),
+                Metadata("Climate Data Operations"),
+                Metadata("PyWPS", "https://pywps.org/"),
+                Metadata("Birdhouse", "http://bird-house.github.io/"),
+                Metadata("PyWPS Demo", "https://pywps-demo.readthedocs.io/en/latest/"),
+            ],
+            inputs=inputs,
+            outputs=outputs,
+            store_supported=True,
+            status_supported=True,
+        )
+
+    def _handler(self, request, response):
+        climdex_input, ci_name, output_path, loglevel = [
+            arg[0] for arg in collect_args(request, self.workdir).values()
+        ]
+
+        log_handler(
+            self,
+            response,
+            "Starting Process",
+            logger,
+            log_level=loglevel,
+            process_step="start",
+        )
+        climdex = get_package("climdex.pcic")
+        base = get_package("base")
```

Comment:
One thing I missed. I don't see `base` used anywhere in the process. Can you omit it?